spec/requests/providers/submitted_applications_controller_spec.rb

```diff
@@ -86,6 +88,24 @@
         expect(response.body).to include("Fake old gateway evidence file (15.7 KB)")
       end
     end
+
+    describe "linked application header" do
+      context "when this is the lead application for no cases" do
+        it { expect(response.body).not_to include("Important") } # Notification header text
```

Comment:
It was because I was trying to prove that the notification banner heading was not displayed, not including `Important` seemed a bit random... not sure it's helped! :)

Reply:
Maybe switch to regex `This application has been linked with .* you submitted it.

Reply:
Switched to use a css matcher for the notification banner